plugins/lookup/github_app_access_token.py

```diff
@@ -154,8 +162,13 @@ def encode_jwt(app_id, private_key_obj, exp=600):
         raise AnsibleError(f"Error while encoding jwt: {e}")
 
 
-def post_request(generated_jwt, installation_id):
-    github_api_url = f'https://api.github.com/app/installations/{installation_id}/access_tokens'
+def post_request(generated_jwt, installation_id, api_base=None):
+    if api_base:
+        base = api_base.rstrip('/')
+        github_api_url = f"{base}/app/installations/{installation_id}/access_tokens"
+    else:
+        github_api_url = f'https://api.github.com/app/installations/{installation_id}/access_tokens'
```

Comment:
This can also be simplified:
```suggestion
    base = api_base.rstrip('/')
    github_api_url = f"{base}/app/installations/{installation_id}/access_tokens"
```